### DISCUSSION TITLE
scVelo-calculated initial cluster is not MKI67 positive cluster, and direction inverts

### DISCUSSION THREAD

**Discussion by hyjforesight** [theislab/scvelo#760] [Q&A]
Hello scVelo,
Could you please tell us how to set the root clusters for running scVelo? My issue is that the initial clusters calculated by scVelo internally are not consistent with the classical Mki67+ and Lgr5+ clusters, which are already known as the stem cell clusters. We want to set these Mki67+ and Lgr5+ clusters as root and rerun scVelo.
Thanks!
Best,
YJ

**Part C is the Mki67+ stem cells. Based on the existing knowledge, C should have 2 differentiating directions: to B and to A. But based on scVelo, it looks like only one direction: A along with C differentiates to B, and cluster 6 looks like the stem cells based on scVelo stream.**

![image](https://user-images.githubusercontent.com/75048821/144104410-b921536e-f9b8-41c1-ad05-64b783ddb77a.png)
![image](https://user-images.githubusercontent.com/75048821/144104843-ae5fe58a-6867-499a-8551-f8f6ae34708b.png)
![image](https://user-images.githubusercontent.com/75048821/144104877-88de6089-e14b-4225-b300-f39da44a3b75.png)
![image](https://user-images.githubusercontent.com/75048821/144104906-a6f7df02-dd3f-4a43-9614-f8f94482b499.png)

**Comment by WeilerP** [comment]
@hyjforesight, it seems to me that your phase portraits simply do not exhibit the expected almond shape (or at least part of it), and, thus, do not offer any/enough information. See _e.g._ #216 or #462.
